src/Extensions/Hosting.Services/ServiceFabricHostBuilder.cs

```diff
@@ -0,0 +1,32 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+using System;
+using System.Fabric;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Hosting;
+
+namespace Microsoft.Omex.Extensions.Hosting.Services
+{
+	/// <summary>
+	/// Wrapper on top of IHostBuilder to propagete proper context type and avoid type registration mistakes
+	/// </summary>
+	public sealed class ServiceFabricHostBuilder<TContext> where TContext : ServiceContext
```

Comment:
I'd suggest `TServiceContext` as in other files

Reply:
Renamed, thanks